packages/ses/src/module-load.js

```diff
@@ -523,7 +528,11 @@ const asyncJobQueue = (errors = []) => {
     setAdd(
       pendingJobs,
       promiseThen(func(...args), noop, error => {
-        arrayPush(errors, error);
+        if (aggregateErrors) {
+          arrayPush(errors, error);
+        } else {
+          throw error;
+        }
```

Comment:
While this implementation is correct AFAICT, I would like to analyze the difference in control flow between this and putting the error in `errors` and shutting down the queue. Is there any work that would be disrupted less/later in that case? 

I don't see any specific differences right now, but I see the potential for things happening in different order between the two.

Reply:
Well, it would be different because we would not throw the original error; we would throw the aggregate-ish `TypeError` (why isn't this an AggregateError, anyway?) instead.